config/default-config.yml

```diff
@@ -137,6 +137,9 @@ general:
 
   # Backup the databases to the database/backups subfolder on halt/suspend/destroy, set to false to disable
   db_backup: false
+     gzip: true
+     exclude:
+       - wordpress-trunk
```

Comment:
Well it is for contributors and usually people keep on the config.yml but doesn't uses it (I saw that from various people).

Reply:
but why would a contributor want to exclude `wordpress-trunk` and why would we want that as the default? IMO this is harmful, `default-config.yml` is what users get out of the box as their initial `config.yml`, it isn't a showcase of every possible option. We shouldn't exclude `wordpress-trunk` unless we can clearly show that this is desirable and necessary for all users

Reply:
Well as contributor I don't need a dump of the WordPress database, so I used it as example.
As not all the VVV users uses the develop version I was thinking that was a good example for a backup not needed.

Reply:
The purpose of the back-up is that you can restore the database when you destroy and recreate the VM. Additionally, some people use the develop template for commercial development, either because they've misunderstood, or to test on bleeding edge.

Unless ***all*** people will find this useful, we should not make it the default for all new users.

Reply:
Well so what website can we configure as default?

Reply:
We don't, that's what documentation and comments are for